### DISCUSSION TITLE
People face

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2303] [General]
Hi, I have a problem with people's faces and fingers, it's always creates something terrible, and the smaller person the worst result. Which style should I use maybe give me a few ideas about the prompt to create a normal human face with the correct arms and fingers.

**Comment by ILikeToasters** [comment]
The further away the worse it will get. You can use tools like improve details in the inpainting and can help if not too bad. If you think about how they train the models and the resolution of the images, real pictures with that low of quality also have terrible faces and hands often as well.

**Comment by barepixels** [comment]
You have to understand the source of the problem.  AI images are generated from noises.  The smaller the area such as hands or face of distance people have less noise information for AI to generate from. We went from 512 (SD1.5) to 768 (SD2.1) and now 1024 (SDXL).  the problem will get less worse as time goes by as resolution (technology) increases.  Even if we have a mega resolution model now.  The home computer won't be able to handle it.  Heck, many people have less than 8 gigs of vram and struggle with SDX.
